rfp-proposal/on-chain-quadratic-funding.md

```diff
@@ -0,0 +1,68 @@
+# On-chain Quadratic Funding
+
+* **Proposer:** [Noc2](https://github.com/Noc2)
+
+## Project Description :page_facing_up: 
+
+CLR, short for [Constrained Liberal Radicalism algorithm](https://blogchains.org/wp-content/uploads/sites/4/2019/04/SSRN-id3243656.pdf), commonly called quadratic funding (QF), is a way to efficiently fund projects in the Web3 ecosystem. The way it works is that users contribute directly to projects, which they value and, in doing so, earn a share of a matching pool for the projects. The *number* and amount of each contribution influences the total amount allocated to a project. This means even a small contribution is valuable and can result in a high matched amount. 
```

Comment:
"earn a share of a matching pool" needs clarification - it reads like contributors getting something, when in fact it's projects getting a share of the pool.